qds_sdk/account.py

```diff
@@ -1,10 +1,64 @@
 """
 The Accounts module contains the base definition for a Qubole account object
 """
-
+import json
 from qds_sdk.resource import SingletonResource
+from qds_sdk.qubole import Qubole
+from qds_sdk.exception import ParseError
+from qds_sdk.util import GentleOptionParser
+from qds_sdk.util import OptionParsingError
+from qds_sdk.util import OptionParsingExit
+import argparse
+
+class AccountCmdLine:
+
+
+    @staticmethod
+    def parsers():
+        argparser = argparse.ArgumentParser(prog="qds.py account",
+                description="Account Creation for Qubole Data Service.")
+        subparsers = argparser.add_subparsers(title = "account operation")
+
+        #Create
+        create = subparsers.add_parser("create",
+                                       help="Create a new account")
+        create.add_argument("--name", dest="name", help="account name")
+        create.add_argument("--location", dest="defloc", help="Default location of S3")
+        create.add_argument("--access", dest = "acc_key", help="access key")
+        create.add_argument("--secret", dest= "secret", help="secret key")
+        create.add_argument("--compute_access", dest="compute_access_key", help="compute access key")
+        create.add_argument("--compute_secret", dest="compute_secret_key", help="compute secret key")
+        create.add_argument('--aws', dest="aws_region", help="aws region")
+        create.add_argument("--previous_account_plan", dest="use_previous_account_plan",choices=["true", "false"], help="either true or false")
+        create.set_defaults(func=AccountCmdLine.create)
+        return argparser
+
+    @staticmethod
+    def run(args):
+        parser = AccountCmdLine.parsers()
+        parsed = parser.parse_args(args)
+        return parsed.func(parsed)
+
+    @staticmethod
+    def create(args):
+        args.level = "free"
```

Comment:
I have only added those options which are present in UI while adding new account and there are some other params which are passed too but are not enter by user like level , compute_type , storage_type and CacheQuotaSizeInGB. For these options default values are already set in js file so I added same values here too.